features/steps/static_primary.py

```diff
@@ -0,0 +1,25 @@
+import json
+import patroni.psycopg as pg
+
+from behave import step, then
+from time import sleep, time
```

Comment:
_F401 'time.time' imported but unused_

```suggestion
from time import sleep
```